mpi-proxy-split/split_process.cpp

```diff
@@ -425,6 +425,8 @@ setLhMemRange()
 }
 
 // Initializes the libraries (libc, libmpi, etc.) of the lower half
+// FIXME: without optimization, lower half initialization segfaults on CentOS
```

Comment:
I think you meant to write:  `_With_ optimization, lower-half initialization ...`